### PR TITLE
Add documentation about "non-zero requests"

### PR DESCRIPTION
**Pull request by misterikkit**
/kind cleanup
/sig scheduling

### CODE REVIEW

File: pkg/scheduler/algorithm/priorities/util/non_zero.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 package util
 
-import "k8s.io/api/core/v1"
+import v1 "k8s.io/api/core/v1"
```
Comment (misterikkit) [inline pkg/scheduler/algorithm/priorities/util/non_zero.go]: Oh no!!! I made the classic VSCode mistake.

Comment (ahg-g) [reply pkg/scheduler/algorithm/priorities/util/non_zero.go]: what's the harm? if many editors add it by default, why not just make it part of the format :)

File: pkg/scheduler/nodeinfo/node_info.go [hunk 2]
```diff
@@ -52,11 +52,14 @@ type NodeInfo struct {
...
 	usedPorts        HostPortInfo
 
-	// Total requested resource of all pods on this node.
-	// It includes assumed pods which scheduler sends binding to apiserver but
-	// didn't get it as scheduled yet.
+	// Total requested resources of all pods on this node. This includes assumed
+	// pods, which scheduler has sent for binding, but may not be scheduled yet.
 	requestedResource *Resource
-	nonzeroRequest    *Resource
+	// Total requested resources of all pods on this node with a minimum value
```
Comment (hex108) [inline pkg/scheduler/nodeinfo/node_info.go]: Would it be better to add an explanation to minimum value: `Total requested resources of all pods on this node with a default value(DefaultMilliCPURequest/DefaultMemoryRequest) applied to each container's CPU and memory requests if they are empty.`

**Comment by egernst** [comment]
This PR looks good to me -- helps clarify the usage.  Anything needed to drive this forward?

**Comment by ahg-g** [comment]
/assign @Huang-Wei 

Can you take a quick look and approve please?

**Review by Huang-Wei — COMMENTED** [review]
/approve
/lgtm

Thanks @misterikkit !

**Pull request merged**